packages/dashmate/README.md

```diff
@@ -242,6 +242,39 @@ To reset a node:
 $ dashmate reset
 ```
 
+### Reindex dashcore chain data
+
+Sometimes you need to make a process of reindex for your core container, in case you imported new private keys, or your core data corrupted. The command `reindex` put your node in the reindex mode.
```

Comment:
```suggestion
Sometimes you need to run a reindex for your core container. This may be necessary if you import new private keys or if your core data has become corrupted. The `reindex` command puts your node into the reindex mode.
```